pkg/hubbub/match.go

```diff
@@ -80,7 +80,7 @@ func preFetchMatch(i GitHubItem, labels []*github.Label, fs []Filter) bool {
 		}
 
 		// This state can be performed without downloading comments
-		if f.TagRegex() != nil && f.TagRegex().String() == "assigned" {
+		if f.TagRegex() != nil && f.TagRegex().String() == "^assigned$" {
```

Comment:
`TagRegex` object is [instantiated](https://github.com/google/triage-party/blob/master/pkg/hubbub/filter.go#L66) with [`regex(...)`](https://github.com/google/triage-party/blob/master/pkg/hubbub/filter.go#L137) method. This wraps tag string with `^` and `$`.  So, the rule should compare with `"^assigned$"`